int32 id
---
bool result
string status_msg